### ISSUE TITLE
vmagent v1.55.0 crashes soon after startup

### ISSUE THREAD

**Issue by umezawatakeshi** [VictoriaMetrics/VictoriaMetrics#1108]
**Describe the bug**

the vmagent crashes soon after startup with the following configuration.

```
global:
  scrape_interval: 30s
  external_labels:
    prometheus: monitoring/vmagent-smallset
scrape_configs:
- job_name: monitoring/kube-state-metrics/0
  honor_labels: true
  kubernetes_sd_configs:
  - role: service
    namespaces:
      names:
      - kube-system
  relabel_configs:
  - action: keep
    source_labels:
    - __meta_kubernetes_service_label_app_kubernetes_io_name
    regex: kube-state-metrics
  - source_labels:
    - __meta_kubernetes_namespace
    target_label: namespace
  - source_labels:
    - __meta_kubernetes_service_name
    target_label: service
  - source_labels:
    - __meta_kubernetes_service_name
    target_label: job
    replacement: ${1}
  - regex: __.*
    action: labelkeep
  - target_label: job
    replacement: kube-state-metrics
  - source_labels:
    - __meta_kubernetes_service_port_name
    regex: (http-metrics|telemetry)
    action: keep
```

**To Reproduce**
run vmagent with below-mentioned command-line flags and above-mentioned configuration file on k8s cluster

**Expected behavior**
start scraping without crash

**Screenshots**

the crash log follows:
```
2021-03-03T07:14:46.733Z        info    VictoriaMetrics/lib/promscrape/scraper.go:91    reading Prometheus configs from "/etc/vmagent/config_out/vmagent.env.yaml"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x8423a6]

goroutine 55 [running]:
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*ScrapeWork).LabelsString(0x0, 0x41243b, 0xc0002322c0)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scrapework.go:118 +0x26
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*ScrapeWork).key(0x0, 0x9, 0xc0007cdb88)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scrapework.go:106 +0x45
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*scraperGroup).update(0xc00083cb40, 0xc00012e400, 0x9, 0x10)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scraper.go:306 +0x16e
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*scrapeConfig).run.func1(0xc0002fe740)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scraper.go:236 +0xe5
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*scrapeConfig).run(0xc0002feb40)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scraper.go:240 +0x153
github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*scrapeConfigs).add.func1(0xc0002fea80, 0xc0002feb40)
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scraper.go:192 +0x59
created by github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape.(*scrapeConfigs).add
        github.com/VictoriaMetrics/VictoriaMetrics/lib/promscrape/scraper.go:190 +0x1dd
```

**Version**
```
vmagent-20210302-195139-tags-v1.55.0-0-g1252ca44d
```

**Used command-line flags**
```
    -httpListenAddr=:8429
    -promscrape.config=/etc/vmagent/config_out/vmagent.env.yaml
    -remoteWrite.maxDiskUsagePerURL=1073741824
    -remoteWrite.tmpDataPath=/tmp/vmagent-remotewrite-data
    -remoteWrite.url=http://vmsingle-vmsingle-smallset.monitoring.svc:8429/api/v1/write
```

**Additional context**

- k8s version: v1.19.7
- vmagent v1.54.0 (`vmagent-20210218-132842-tags-v1.54.0-0-g03ebc028f`) seems to work fine with same command-line flags and configuration file.
- This vmagent is managed by VictoriaMetrics operator v0.9.0
the VMAgent resource is as follows:
```
apiVersion: operator.victoriametrics.com/v1beta1
kind: VMAgent
metadata:
  name: vmagent-smallset
  namespace: monitoring
spec:
  image:
    tag: v1.55.0
  nodeScrapeSelector:
    matchLabels:
      smallset: "true"
  podScrapeSelector:
    matchLabels:
      smallset: "true"
  probeSelector:
    matchLabels:
      smallset: "true"
  serviceScrapeSelector:
    matchLabels:
      smallset: "true"
  remoteWrite:
  - url: http://vmsingle-vmsingle-smallset.monitoring.svc:8429/api/v1/write
  replicaCount: 1
  resources:
    requests:
      cpu: 100m
      memory: 200Mi
  securityContext:
    runAsGroup: 10000
    runAsUser: 10000
```

**Comment by umezawatakeshi** [comment]
oops, seems to be duplicate #1107

**Comment by valyala** [comment]
The bug must be fixed in the commit 7906316741eb7deed2d0d7fb90ba50505ba37de3

**Comment by valyala** [comment]
Closing this issue as a duplicate for #1107
